src/interface/src/app/services/local-storage.service.ts

```diff
@@ -0,0 +1,41 @@
+import { Injectable } from '@angular/core';
+import { Params } from '@angular/router';
+import { RedirectData } from '@services/redirect.service';
+
+export abstract class BaseStorageService<T> {
```

Comment:
nit: I'm inclined to change this to BaseLocalStorageService or something, just to clarify what kind of storage it's handling.

Reply:
Sounds good! I have a follow up pr, will update it there